content/doc/book/pipeline/syntax.adoc

```diff
@@ -417,26 +417,26 @@ shown below.
 
 `always`:: Run the steps in the `post` section regardless of the completion
 status of the Pipeline's or stage's run.
-`changed`:: Only run the steps in `post` if the current Pipeline's or stage's
+`changed`:: Only run the steps in `post` if the current Pipeline's
 run has a different completion status from its previous run.
-`fixed`:: Only run the steps in `post` if the current Pipeline's or
-stage's run is successful and the previous run failed or was unstable.
+`fixed`:: Only run the steps in `post` if the current Pipeline's
+run is successful and the previous run failed or was unstable.
 `regression`:: Only run the steps in `post` if the current Pipeline's
-or stage's run's status is failure, unstable, or aborted and the previous run
+or status is failure, unstable, or aborted and the previous run
 was successful.
-`aborted`:: Only run the steps in `post` if the current Pipeline's or stage's
+`aborted`:: Only run the steps in `post` if the current Pipeline's
 run has an "aborted" status, usually due to the Pipeline being manually aborted.
 This is typically denoted by gray in the web UI.
 `failure`:: Only run the steps in `post` if the current Pipeline's or stage's
 run has a "failed" status, typically denoted by red in the web UI.
 `success`:: Only run the steps in `post` if the current Pipeline's or stage's
 run has a "success" status, typically denoted by blue or green in the web UI.
-`unstable`:: Only run the steps in `post` if the current Pipeline's or stage's
+`unstable`:: Only run the steps in `post` if the current Pipeline's
```

Comment:
How does this square with the `success` block running if the stage is successful but the pipeline is not?

IOW, it doesn't seem to make sense that success takes into account the stage's status and unstable does not.  If the pipeline is unstable but the stage is success, this updated description for unstable seems like it's going to override success's using the stage's status and void that evaluation for success also, effectively degenerating success to only evaluating the pipeline's status.

But that said, it seems very disingenuous to take the success/unstable processing paths away from stages based on what other stages do.  I'm having a hard time imagining any use-case for this.  Stages should be independent of each other and have full control over their processing without interference from other stages.